src/runtime/propensity-test.js

```diff
@@ -166,4 +172,60 @@ describes.realWin('Propensity', {}, env => {
       expect(propensityScore.body.result).to.equal(42);
     });
   });
+
+  it('should convert all propensity events to analytics events', () => {
+    let eventSent = null;
+    const params = /** @type {JsonObject} */ ({
+      'source': 'user-action',
```

Comment:
lets remove is_active since we plan to clarify the API anyway.
And let's use a source that's more believable like  'email'

Test code can serve as an example of API usage so lets be careful, although I recognize it does not make any difference to the test logic.

Reply:
Ok